stdlib/collections/__init__.pyi

```diff
@@ -298,22 +300,30 @@ class OrderedDict(dict[_KT, _VT], Reversible[_KT], Generic[_KT, _VT]):
 class defaultdict(dict[_KT, _VT], Generic[_KT, _VT]):
     default_factory: Callable[[], _VT] | None
     @overload
-    def __init__(self, **kwargs: _VT) -> None: ...
+    def __init__(self: defaultdict[_KT, _VT]) -> None: ...
```

Comment:
Why does this overload have an explicit self type?

Reply:
For consistency with `dict`.